db/seeds.rb

```diff
@@ -201,13 +201,30 @@
 
 Rails.logger.info "7. Creating Broadcasts"
 
+# TODO: [@thepracticaldev/delightful] Remove this once we have launched welcome notifications.
```

Comment:
I added this `TODO` specifically for our team. This is something I have found really effective when working on longer features, and in the past, I've linked to a GitHub issue directly. This makes it _super_ easy to make sure that, when you release a feature or major change, you remember to do certain things that you would have otherwise easily forgotten along the way. All you have to do is search for one string in the codebase 😸